server/views/referrals/show/risksAndNeeds/offenceAnalysis.njk

```diff
@@ -0,0 +1,15 @@
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+{% extends "./partials/layout.njk" %}
+
+{% block risksAndNeedsContent %}
+  {{ govukInsetText({
+    classes: "govuk-!-margin-top-0",
+    text: importedFromText,
+    attributes: {
+      "data-testid": "imported-from-text"
+    }
+  }) }}
```

Comment:
Will this be on every risks and needs page? If so, would it make sense to put it in the layout?

Reply:
Yeah good shout, I think I'll do this in #440.